resources/local/config.toml

```diff
@@ -206,6 +206,16 @@ trie_responses = 0
 finalized_approvals_requests = 1
 finalized_approvals_responses = 0
 
+# Threshold for when the networking considers itself sufficiently connected.
+#
+# Some processes will not begin (or fail) unless this threshold is reached.
+[network.bootstrap_threshold]
+# A percentage of known addresses has been reached.
+known_address_connection_percentage = 0.5
```

Comment:
In the config, we already have `saturation_limit_percent` which is an integer, and `acceleration_ftt` which is a percentage defined as a TOML array.  I think we should avoid using a float here, and instead use an integer (i.e. change the `0.5` to `50`.